app/controllers/miq_policy_controller/miq_actions.rb

```diff
@@ -301,7 +300,7 @@ def action_build_alert_choices
   # Build the categories/tags tree node
   def action_build_cat_tree(cats)
     name = 'action_tags'
-    @cat_tree = TreeBuilderMiqActionCat.new(name, name, @sb, "#{current_tenant.name} Tags")
+    @cat_tree = TreeBuilderMiqActionCat.new(name + '_tree', name, @sb, "#{current_tenant.name} Tags", true)
```

Comment:
prefer `"#{name}_tree"` over `+` .. but in this case, just do `'action_tags_tree', 'action_tags'` :)